### PR TITLE
Pull from upstream repository, version 2.3.2

### PR DESCRIPTION
**Pull request by markbreedlove**
Pull commits from upstream `saltstack-formulas/nginx-formula`, tag `v2.2.1`.

Our changes to `nginx/ng/map.jinja` for upgrading the Nginx version have been abandoned.

I have moved our changes to `nginx/ng/config.sls`, to `nginx/config.sls`, with a minor alteration to correct the `source` of the `file.managed` state in `create_extra_config_{{ fname }}`.

### CODE REVIEW

File: nginx/config.sls [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+create_extra_config_{{ fname }}:
+  file.managed:
+    {{ sls_block(settings.pop('opts', {})) }}
+    - name: {{ nginx.server.get('conf_include_dir', '/etc/nginx/conf.d') }}/{{ fname }}.conf
+    - source: salt://nginx/files/default/nginx.conf
+    - template: jinja
+    - makedirs: True
+    - context:
+        config: {{ settings|json(sort_keys=False) }}
+{% endfor %}
```
Comment (markbreedlove) [inline nginx/config.sls:54]: The block above is the one that I copied from `nginx/ng/config.sls` and modified slightly to fix the `source` attribute of `file.managed`.

**Comment by blarghmatey** [comment]
Looks good, but before we merge we'll need to update our pillar data accordingly. Might want to wait until we get the separate masters set up so we can pin prod and non-prod separately.

**Comment by markbreedlove** [comment]
I pulled more changes from the upstream, so now we're looking at version 2.3.2 of `saltstack-formulas/nginx-formula`.

**Comment by blarghmatey** [comment]
:+1:

**Pull request merged**